README.md

```diff
@@ -3,9 +3,38 @@
 [![Build status](https://badge.buildkite.com/c9edf020a4aec976f9835e54751cc5409d843adbb66d043bd3.svg?branch=main)](https://buildkite.com/ankitects/anki-ci)
 
 This repo contains the source code for the computer version of
-[Anki](https://apps.ankiweb.net).
+[Anki](https://apps.ankiweb.net) for people who would like to help improve the code.
```

Comment:
I don't feel this is an improvement I'm afraid - people may want access to the code without wanting to improve it.